.github/workflows/unit-tests.yaml

```diff
@@ -261,6 +261,22 @@ jobs:
         jira-token: ${{ secrets.JIRA_TOKEN }}
         directory: 'shell-test-output'
 
+  openshift-ci-unit-tests:
```

Comment:
not a blocker: does this job need to be added to `slack-on-unit-tests-failure`?

Reply:
> not a blocker: does this job need to be added to `slack-on-unit-tests-failure`?

Yes. Thanks.